manual.txt

```diff
@@ -9,6 +9,11 @@
     and each of them can be given any name. What is more, in each scenario many processing methods can be used in any order,
     and one processing method can be used more than once in the single scenario.
 
+    As the result, for each scenario user receives .csv files which contains:
+    * extracted features from processed signal
+        - only if at least one feature was specified in the scenario (file placed in ./results directory)
+    *  signal after all modifications and running processing methods (file placed in ./signals directory).
```

Comment:
Myślę też, że warto zrobić tak, że wyniki są w jednym folderze - a w nim poszczególne w pod folderach.
Proponuje taki układ: features do ./results/features, a sygnał do ./results/signals

Dzięki temu:
- użytkownik nie dostanie, nowych sygnałów w folderze w którym kazaliśmy mu wstawić swój sygnał.
- użytkownik będzie miał tylko jedno miejsce gdzie ma szukac wyników - a nie dwa,
- rozwiąże to problem w repo, bo caly plik ./results mozemy po prostu ignorowac dzieki temu.